packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -17,6 +17,7 @@ import { Predeploys } from "src/libraries/Predeploys.sol";
 import { Types } from "scripts/Types.sol";
 import { Vm } from "forge-std/Vm.sol";
 import { ISystemConfigV0 } from "scripts/interfaces/ISystemConfigV0.sol";
+import { console2 as console } from "forge-std/console2.sol";
```

Comment:
Dead code?

Reply:
No, it's restoring this check: 

https://github.com/ethereum-optimism/optimism/blob/remove-mcp/packages/contracts-bedrock/scripts/ChainAssertions.sol#L132-L135